atlas/commands/command.go

```diff
@@ -17,8 +18,11 @@ type Command interface {
 // GetCommandSet returns a mapping between command names and commands
 func GetCommandSet() map[string]Command {
 	cmdBootstrap := bootstrap.Bootstrap{}
+	cmdUpdate := update.Update{}
+
 	return map[string]Command{
 		cmdBootstrap.GetName(): cmdBootstrap,
```

Comment:
Inside func GetCommandNames please add the following snipped so it shows the update as well: 
    ```
    cmdUpdate := update.Update{}
	return []string{cmdBootstrap.GetName(), cmdUpdate.GetName()}
    ```